src/dackar/utils/mbse/customMBSEparser.py

```diff
@@ -0,0 +1,180 @@
+# Copyright 2024, Battelle Energy Alliance, LLC  ALL RIGHTS RESERVED
+
+"""
+Created on February, 2024
+
+@author: mandd
+"""
+
+# External Imports
+import pandas as pd
+import logging
+
+logger = logging.getLogger(__name__)
+
+class customMBSEobject(object):
+    """
+        Class designed to process the a custom MBSE model from file.
+    """
+    def __init__(self, nodes_filename, edges_filename):
+        """
+        Initialization method for the custom MBSE model class
+
+        Args:
+
+            nodes_filename: file, file in .csv format containing all nodes 
+            edges_filename: file, file in .csv format containing all edges 
+
+        Returns:
+
+            None
+        """
+        self.nodes_filename = nodes_filename
+        self.edges_filename = edges_filename
+        self.entities = []
+        self.list_IDs = []
+
+        self.allowed_node_types = ['entity']
+        self.allowed_edge_types = ['link','composition','support'] # to be developed: 'opm_instance'
+
+        self.allowed_node_cols = ['nodeID','label','ID','type','description']
+        self.allowed_edge_cols = ['sourceNodeId','targetNodeId','type','medium','description']
```

Comment:
Do we want to keep camelBack code style? We have discussed it before, and we agreed we would like to make an exception Anomaly Detection Module. Do you want to extend the standard to other modules?